configman/option.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         not_for_definition=False,
         reference_value_from=None,
         secret=False,
+        change_flag=False,
```

Comment:
What's the point of this? 
It gets set in runtime by the various `if val != self.value` things anyway.